### PR TITLE
BMR-33: ADD ROOM - MAKU 

### PR DESCRIPTION
**Pull request by ma-za-kpe**
## SCREENSHOTS (Light mode)
new

### CODE REVIEW

**Comment by esthcarelle** [comment]
@ma-za-kpe  could you please add screenshots?

File: ui/admin/src/main/java/com/amalitech/admin/room/AddRoomScreen.kt [hunk 1]
```diff
@@ -0,0 +1,368 @@
...
+import androidx.compose.ui.text.input.ImeAction
+import androidx.compose.ui.text.input.KeyboardType
+import androidx.compose.ui.text.input.VisualTransformation
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.constraintlayout.compose.ConstraintLayout
+import androidx.constraintlayout.compose.Dimension
+import androidx.lifecycle.compose.collectAsStateWithLifecycle
```
Comment (esthcarelle) [inline ui/admin/src/main/java/com/amalitech/admin/room/AddRoomScreen.kt:60]: Could you please remove unused imports and format codes?

File: ui/admin/src/main/java/com/amalitech/admin/room/AddRoomScreen.kt [hunk 2]
```diff
@@ -0,0 +1,368 @@
...
+                    .constrainAs(add_photos) {
+                        top.linkTo(parent.top, 48.dp)
+                        start.linkTo(parent.start, 16.dp)
+                        end.linkTo(parent.end, 8.dp)
+                        width = Dimension.fillToConstraints
+                    },
+                color = MaterialTheme.colorScheme.onSurfaceVariant,
+                textAlign = TextAlign.Start,
+                fontWeight = FontWeight.Light,
+                fontSize = 16.sp
```
Comment (esthcarelle) [inline ui/admin/src/main/java/com/amalitech/admin/room/AddRoomScreen.kt:94]: Could you please use dimensions that are already defined in the project?

File: ui/admin/src/main/java/com/amalitech/admin/room/AddRoomScreen.kt [hunk 3]
```diff
@@ -0,0 +1,368 @@
...
+                    Icon(
+                        imageVector = Icons.Filled.Add,
+                        contentDescription = null,
+                        modifier = Modifier.size(50.dp),
+                        tint = MaterialTheme.colorScheme.inverseOnSurface
+                    )
+                }
+            }
+
+            val list = listOf(
```
Comment (esthcarelle) [inline ui/admin/src/main/java/com/amalitech/admin/room/AddRoomScreen.kt:132]: What is this list A,B,C about?

Comment (ma-za-kpe) [reply ui/admin/src/main/java/com/amalitech/admin/room/AddRoomScreen.kt:132]: im still workin on it, havent requested review

**Review by bensek — APPROVED** [review]

**Review by esthcarelle — APPROVED** [review]

**Comment by esthcarelle** [comment]
@ma-za-kpe are the changes you were talking about done?

**Comment by esthcarelle** [comment]
@ma-za-kpe  conflicts

**Comment by ma-za-kpe** [comment]
> @ma-za-kpe conflicts

Haven't finished the task yet, please don't merge @esthcarelle  and @bensek

**Comment by ma-za-kpe** [comment]
Is there anything else I should add as we wait for the API? @esthcarelle  and @bensek

**Comment by bensek** [comment]
This is fine we can merge it. You should look into creating the user flow for the Admin section. e.g SplashScreen -> Dashboard -> Rooms Screen -> Add Room.

**Comment by ma-za-kpe** [comment]
> This is fine we can merge it. You should look into creating the user flow for the Admin section. e.g SplashScreen -> Dashboard -> Rooms Screen -> Add Room.

Can this be in task form, because I don't seem to have any more tasks?

**Pull request merged**